net/haproxy/Portfile

```diff
@@ -52,6 +56,8 @@ build.target        TARGET=osx
 pre-build {
     build.args      LD="${configure.cc}"\
                     LDFLAGS="${configure.ldflags}" \
+                    SSL_INC=[openssl::include_dir] \
+                    SSL_LIB=[openssl::install_area] \
```

Comment:
Per documentation, `SSL_LIB` should be pointing at the `.../openssl-.../lib` directory, so this should be set to `[openssl::lib_dir]`.

Reply:
Fix